SwrveSDKTest/Podfile

```diff
@@ -0,0 +1,13 @@
+target 'SwrveSDKTest' do
+  pod 'SwrveSDKCommon', :path => '../'
+  pod 'SwrveConversationSDK', :path => '../'
+  pod 'SwrveSDK', :path => '../'
+  #pod 'SwrveSDK', '~> 4.5.0'
```

Comment:
This would be handy to test on release. We can run all the tests against the latest version.